### ISSUE TITLE
Support APK PCKS7 signature SHA1 digests

### ISSUE THREAD

**Issue by g-k** [mozilla-services/autograph#156]
Some APKs need a SHA1 digest to match their cert, but [we have this hardcoded to SHA256
](https://github.com/mozilla-services/autograph/blob/master/signer/apk/apk.go#L146-L147
).

We can copy the digest option for XPIs to support this (and default to SHA256):

```json
"options": {
                     "pkcs7_digest": "SHA256" // or "SHA1"
                },
```

https://github.com/mozilla-services/autograph/tree/master/signer/xpi#2signature-request

However, differing from the XPI option it won't be required and we'll allow it to be blank (meaning default/SHA256).

This would also require a change in signingscript possibly by adding a new format like `autograph_apk_sha1`. If we need support from autograph edge then we should add a way to pass the option through from there too.

**Comment by jvehent** [comment]
> Some APKs need a SHA1 digest to match their cert

Do they? If android doesn't require the hash algorithm of the pkcs7 sigalg to match the signing cert sigalg, then enforcing SHA256 here is preferable.

**Comment by g-k** [comment]
Yes, per @JohanLorenzo on irc to avoid breaking compat w/ old android versions that don't support SHA2+: https://bugzilla.mozilla.org/show_bug.cgi?id=1332916

**Comment by g-k** [comment]
Fixed in #162.
